client/src/modules/user/UserProfile/UserProfile.jsx

```diff
@@ -1,29 +1,46 @@
-import React, { PureComponent } from 'react';
+import React, { PureComponent, createRef } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import { FormattedDate, FormattedMessage, FormattedTime } from 'react-intl';
 import classNames from 'classnames';
+import isEmpty from 'validator/lib/isEmpty';
+import isLength from 'validator/lib/isLength';
+import _trim from 'lodash/trim';
```

Comment:
can we use js trim instead of `lodash`? Is `lodash` trim better than the native js method? Przemo said that if possible, it is better to use native methods.